### PR TITLE
feat: use eslintv9 compatible beta version of eslint-config-crowdstrike

### PR DESCRIPTION
**Pull request by Charlie-Crowdstrike**
* use eslintv9 compatible beta version of eslint-config-crowdstrike
* remove eslintv8 compatibility shims and packages
* bundle eslint-config-crowdstrike directly

After this change, this repo is ready to release `4.0.0`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,20 +1,13 @@
 'use strict';
 
-const jsonFiles = require('eslint-plugin-json-files');
+const crowdstrikeConfig = require('eslint-config-crowdstrike');
 const globals = require('globals');
-const { FlatCompat } = require('@eslint/eslintrc');
```
Comment (Charlie-Crowdstrike) [inline index.js:5]: this is messy due to me alphabetizing. TLDR: nuked the eslintv8 compatibility packages

**Comment by Charlie-Crowdstrike** [comment]
@kellyselden should we actually cut a major release of https://github.com/CrowdStrike/eslint-config-crowdstrike instead of a beta version, so the next release of https://github.com/CrowdStrike/eslint-config-crowdstrike-node can be released with all major versions in its `dependencies`?

**Comment by kellyselden** [comment]
We can do another round of betas before doing the real ones.

**Comment by kellyselden** [comment]
+ eslint-config-crowdstrike-node@4.0.0-beta.2

**Pull request merged**